deploy/cloud/modules/ec/main.tf

```diff
@@ -0,0 +1,67 @@
+locals {
+  version                          = var.stack_version
+  region                           = var.region
+  name_prefix                      = var.deployment_name_prefix
+  deployment_template              = var.deployment_template
+  es_docker_image                  = lookup(var.docker_image, "elasticsearch", "")
+  es_docker_image_tag_override     = lookup(var.docker_image_tag_override, "elasticsearch", "")
+  kibana_docker_image              = lookup(var.docker_image, "kibana", "")
+  kibana_docker_image_tag_override = lookup(var.docker_image_tag_override, "kibana", "")
+  apm_docker_image                 = lookup(var.docker_image, "apm", "")
+  apm_docker_image_tag_override    = lookup(var.docker_image_tag_override, "apm", "")
```

Comment:
Do we really have a usecase of overriding these?

Reply:
why do we even need the apm component?

Reply:
The default `manual` deployment on the cloud includes the `Integrations server` component.
We are installing the `Integrations server` component, which was [introduced](https://www.elastic.co/guide/en/cloud/current/ec-release-notes-2022-02-03.html) in version 8.x and [consists](https://www.elastic.co/guide/en/apm/guide/8.11/apm-quick-start.html) of APM and Fleet Server. Fleet is necessary to centrally manage all agents, integrations, and APM is part of Integrations server component.